app/views/posts/index.html.erb

```diff
@@ -1,3 +1,54 @@
-<h1>Input your favourite RSS feed</h1>
 
-<%= render 'form' %>
+<div class="container">
+
+<div class="page-header">
+<h1>Share or Schedule</h1>
+</div>
+
+<div class="row row-flex row-flex-wrap">
+
+	<% @urls.find_each do |url| %>
+       <% @feed = Feedjira::Feed.fetch_and_parse url.url %>
+        <% @entries = @feed.entries %>
```

Comment:
@donburi 
.entries is giving me an 'undefined method `entries' for 200:Fixnum' is this a feedjia method?

It's also kind of confusing that we have @entries and entries which are different.